src/operator/race-static.ts

```diff
@@ -0,0 +1,16 @@
+import {Observable} from '../Observable';
+import {RaceOperator} from './race-support';
+import {ArrayObservable} from '../observable/fromArray';
+
+/**
+ * Returns an Observable that mirrors the first source Observable to emit an item.
+ * @param {...Observables} ...observables sources used to race for which Observable emits first.
+ * @returns {Observable} an Observable that mirrors the output of the first Observable to emit an item.
+ */
+export function race<T>(...observables: Array<Observable<any>>): Observable<T> {
+  if (observables.length === 1) {
```

Comment:
@justinwoo could you add support for a single `Array<Observable>` argument also? Similar to [combineLatest-static](https://github.com/ReactiveX/RxJS/blob/master/src/operator/combineLatest-static.ts#L35), this would bring it in line with the [old signature](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/amb.md) of Observable.amb().